cmake/external/glog.cmake

```diff
@@ -55,7 +55,6 @@ ExternalProject_Add(
 ADD_LIBRARY(glog STATIC IMPORTED GLOBAL)
 SET_PROPERTY(TARGET glog PROPERTY IMPORTED_LOCATION ${GLOG_LIBRARIES})
 ADD_DEPENDENCIES(glog extern_glog gflags)
-LINK_LIBRARIES(glog gflags)
```

Comment:
This `LINK_LIBRARIES` statement will enable the link of `gflags` and `glog` to all executables. I add the statement back, because I found many modules use `glog` but not add `glog` to their `DEPS` in `cc_library`.